tests/migrator/test_journals.py

```diff
@@ -172,9 +170,13 @@ def test_note(app):
                 The reports are catalogued individually and retrievable in the catalogue.
                 </subfield>
             </datafield>
+            <datafield tag="866" ind1=" " ind2=" ">
+                <subfield code="a">note1</subfield>
+                <subfield code="b">note2</subfield>
+            </datafield>
             """,
             {
-                "note": "The reports are catalogued individually and retrievable in the catalogue.",
+                "note": "The reports are catalogued individually and retrievable in the catalogue. / note1 / note2",
```

Comment:
👍 
we chose to go with this solution, would you have any suggestions @ntarocco how to solve it  better?

Reply:
Both `935` and `866` they are mapped to the `note` field, right? 
I am wondering if we add a `\n` instead: if in the form this is a `textarea`, new lines are `\n` I guess... But to be tested both when editing and when visualizing in the details page.

Reply:
In the textarea the `\n` works correctly but in the notes tab in the details page it shows all notes in the same line. It probably needs a javascript function to replace the `\n` with `<br />`